### PR TITLE
Fix gorm types

### PR DESCRIPTION
**Pull request by anatheka**
This PR fixes the following gorm types
- google.protobuf.Timestamp -> datetime
- google.protobuf.Duration -> interval

### CODE REVIEW

File: api/orchestrator/orchestrator.proto [hunk 1]
```diff
@@ -749,7 +749,8 @@ message Category {
   repeated Control controls = 4 [
     (tagger.tags) =
       "gorm:\"foreignKey:category_name,category_"
-      "catalog_id;references:name,catalog_id;constraint:OnDelete:CASCADE\"",
+      "catalog_id;references:name,catalog_id;constraint:OnDelete:CASCADE\""
+,
```
Comment (lebogg) [inline api/orchestrator/orchestrator.proto:753]: New weird comme placement?

File: api/orchestrator/orchestrator.proto [hunk 2]
```diff
@@ -793,7 +794,8 @@ message Control {
     (tagger.tags) =
       "gorm:\"foreignKey:parent_control_id,parent_control_"
       "category_name,parent_control_category_catalog_id;"
-      "references=id,category_name;category_catalog_id\"",
+      "references=id,category_name;category_catalog_id\""
+,
```
Comment (lebogg) [inline api/orchestrator/orchestrator.proto:798]: See above

**Review by lebogg — APPROVED** [review]
LGTM

File: api/orchestrator/orchestrator.proto (hunk 1 above)
Comment (anatheka) [inline api/orchestrator/orchestrator.proto:753]:
```suggestion
      "catalog_id;references:name,catalog_id;constraint:OnDelete:CASCADE\"",
```

File: api/orchestrator/orchestrator.proto (hunk 2 above)
Comment (anatheka) [inline api/orchestrator/orchestrator.proto:798]:
```suggestion
      "references=id,category_name;category_catalog_id\"",
```

**Pull request merged**